pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -296,6 +297,14 @@ type TidbClusterStatus struct {
 	Conditions []TidbClusterCondition `json:"conditions,omitempty"`
 }
 
+type BinlogNodeStatus struct {
+	NodeID      string `json:"nodeId"`
+	Addr        string `json:"host"`
```

Comment:
Can we unify these names to `addr` or `host`? (or you can just let it go...

Reply:
let it go!